test/common/test_centroid.cpp

```diff
@@ -167,26 +173,33 @@ TEST (PCL, compute3DCentroidDouble)
   std::vector<int> indices;
   PointXYZ point;
   PointCloud<PointXYZ> cloud;
-  Eigen::Vector4d centroid;
+  // set centroid to any value (::Ones is just used for convenience)
+  Eigen::Vector4d centroid = Eigen::Vector4d::Ones ();
```

Comment:
Same comment as before.

Reply:
I thought the vector would be zero if uninitialized, but turns out that's not true. So I've removed the initialization.